test/shared/src/main/scala/zio/test/mock/Method.scala

```diff
@@ -0,0 +1,3 @@
+package zio.test.mock
+
+trait Method[+A, B]
```

Comment:
Can this be contravariant in `B`?

Reply:
I remember it blowing up in some places when I did that, but perhaps with some tweaks here and there it could work. Leaving this open to investigate in future.

Reply:
OK, I just tried it and it works now. Probably during the hackathon I had some types wrongly set in `expects` return type. Also, it seems like the type inference problem is gone now. :boom:

Reply:
Variance for the win!